src/docs/product/relay/options.mdx

```diff
@@ -199,6 +199,42 @@ See also `cache.envelope_expiry`.
 
 Interval for evicting outdated project configs from memory.
 
+## Spooling
+
+These settings control on-disk spooling. Currently only spooling for incoming envelopes is supported.
+
+`spool.envelopes.path`
+
+: _String, default: null_
+
+  The path to the spool DB file, which is used to persist the envelopes to disk.
```

Comment:
```suggestion
These settings control on-disk spooling. Currently, only spooling for incoming envelopes is supported.

`spool.envelopes.path`

: _String, default: null_

  The path to the spool DB file, which is used to persist the envelopes to disk.
```
Not familiar with this concept, but wan to double check that "persist" is the correct word to be used here?

Reply:
Thanks! Changed it to `save` , which precisely describes what's going on.